Jenkinsfile

```diff
@@ -87,6 +92,27 @@
     }
 }
 
+def startLogCatCollector() {
+    sh '''rm -f logcat.txt
+    adb logcat -c
+    adb logcat > "logcat.txt" &
+    '''
+}
+
+def stopLogCatCollector(String pid, boolean archiveLog) {
+    sh '''jobs
+       kill %1'
```

Comment:
I suppose `jobs` is just for debugging purposes, right?
if you use double quotes you will be able to use the `pid` argument